### ISSUE TITLE
NullReferenceException

### ISSUE THREAD

**Issue by 0ptera** [mspielberg/dv-zsounds#2]
I get NullReferenceExceptions trying to force all "DB" skinned sh282 to use the BR52 whistle.
Even manually applying the sound throws the nullref.

reloadconfig looks like it properly loads everything:

![20220423090008_1](https://user-images.githubusercontent.com/8436934/164884054-e29395c4-62f6-4f35-9493-25b023a06130.jpg)
![20220423090003_1](https://user-images.githubusercontent.com/8436934/164884058-9dd1fdf6-465b-426b-9229-19c7e9b427dc.jpg)

```json
{
    "version": 1,
    "hooks": [
        {
            "type": "AddRule",
            "path": "loco_sh282",
            "rule": {
                "type": "If",
                "property": "SkinName",
                "value": "DB",
                "rule": "loco_br52"
            }
        }
    ],
    "rules": {
        "loco_br52": {
            "type": "AllOf",
            "sounds": [ "br52_whistle" ]
        }
    },
    "sounds": {
        "br52_whistle": { "type": "Whistle", "filename": "BR52_Pfeife_improved.ogg" }
    }
}
```
zsounds-config.json and BR52_Pfeife_improved.ogg are in \Mods\ZSounds\BR52\
the skin path is \Mods\SkinManagerMod\Skins\loco_steam_H\DB\

**Comment by mspielberg** [comment]
I'm not able to reproduce on my end. Could you post a log showing the exception backtrace?

**Comment by 0ptera** [comment]
[Player.log](https://github.com/mspielberg/dv-zsounds/files/8549067/Player.log)

**Comment by mspielberg** [comment]
Should be fixed in v1.4.0 by https://github.com/mspielberg/dv-zsounds/commit/7e3e8b60178aae1996aa8781e3c1e622558456db.

**Comment by 0ptera** [comment]
It is, thanks.
